resources/v2/02_subjects.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE subjects
+(
+    id SERIAL NOT NULL PRIMARY KEY,
+    name text NOT NULL,
+    kind text NOT NULL,
+    
+    created_at timestamp
+    WITH time zone DEFAULT NOW(),
+    updated_at timestamp
+    WITH time zone,
+    deleted_at timestamp
+    WITH time zone
+);
+
+CREATE INDEX subjects_name ON USING btree (name)
+CREATE INDEX subjects_kind ON USING btree (kind)
```

Comment:
Do you require this ? Can you explain about the use case ?

Reply:
I mean won't we require `indexing` for `name` or `kind` ie. looking up for the subjects via the same fields ?